launchUI.py

```diff
@@ -669,7 +666,7 @@ def leave(event):
 def clickTile(ID):
     widgets["fuConfigPannel"].config(text='Tile '+str(ID)+' functional units')
     widgets["xbarConfigPannel"].config(text='Tile '+str(ID)+' crossbar outgoing links')
-    widgets["xbarConfigPannel"].grid(columnspan=4, row=7, column=0,rowspan=2,sticky="nsew")
+    widgets["xbarConfigPannel"].grid(columnspan=4, row=7, column=0, rowspan=2, sticky="nsew")# After clicking the tile, the pannel will fill all directions

```

Comment:
nit: move the comment above the corresponding code

Reply:
I mean for all the comments, they should be above the corresponding code. Please make sure all of them are resolved.